plantswap/components/Results/results.module.scss

```diff
@@ -2,176 +2,92 @@
-//     position: absolute;
-//     background-color:#d9d9d9;
-//     border-radius: 100px;
-//     padding: 5px 15px;
-//     right: 80px;
-//     top: 20px;
-//   }
-//   ::before {
-//     content: "Match";
-//     position: absolute;
-//     right: 20px;
-//     top: 25px;
-//   }
-// }
-.second {
-  position: relative;
-}
-.second::after {
-  content: "2";
-  position: absolute;
-  background-color:#d9d9d9;
-  border-radius: 100px;
-  padding: 5px 15px;
-  right: 80px;
-  top: 20px;
-}
-.second::before {
-  content: "Match";
-  position: absolute;
-  right: 20px;
-  top: 25px;
-}
-
-.third {
-  position: relative;
-}
-.third::after {
-  content: "3";
-  position: absolute;
-  background-color:#d9d9d9;
-  border-radius: 100px;
-  padding: 5px 15px;
-  right: 80px;
-  top: 20px;
-}
-.third::before {
-  content: "Match";
-  position: absolute;
-  right: 20px;
-  top: 25px;
-}
-
-
-@keyframes fadeIn {
-  from {
-    transform: translateY(100%);
-  }
-
-  to {
-    transform: translateY(0);
+  &__count {
+    // margin-left: auto;
   }
 }
 
 .result {
   display: grid;
-  grid-template-columns: 2fr 4fr 3fr;
-  grid-template-rows:  .5fr .5fr .5fr;
+  // grid-template-columns: 2fr 4fr 3fr;
+  // grid-template-rows: 1fr auto auto;
   width: 100%;
   border-radius: var(--border-radius);
   color: var(--text-color);
   background-color: var(--color-white);
-  padding: 20px;
+  padding: 1rem;
+  gap: 0.25rem 1rem;
+  grid-template-rows: 1fr auto auto;
+  grid-template-columns: auto 1fr auto;
+
+  grid-template-areas:
+    "img x rank"
+    "img name button"
+    "img latin button";
 
   &__grade {
+    grid-area: grade;
     font-size: 14px;
     font-weight: 400;
-    grid-row: 2;
     align-self: start;
-    grid-column: 3;
     justify-self: end;
-   
   }
 
   &__name {
-    grid-column: 2;
-    grid-row: 2;
-    font-size: 16px;
-    font-weight: 700;
+    grid-area: name;
     align-self: end;
-    padding-left: 20px;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+    font-weight: 400;
   }
 
   &__latin {
-    color: var(--color-black);
+    grid-area: latin;
+    color: var(--color-dark-grey);
     font-weight: 400;
-    font-size: 14px;
-    grid-column: 2;
-    grid-row: 3;
-    align-self: end;
-    padding-left: 20px;
-    
+
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+    align-self: flex-end;
+    // padding-left: 20px;
   }
 
   &__btn {
-    height: 40px; 
-    width: 120px;
-    background-color: #d9d9d9;
-    border-radius: 20px;
-    grid-column: 3;
-    grid-row: 3;
+    grid-area: button;
     align-self: end;
-    justify-self: end;
-    font-size: 14px;
-
-
-    //Button content
-    display: flex;
-    justify-content: center;
-    align-items: center;
   }
+
   &__img {
-    grid-column: 1; 
-    grid-row:  span 4;
-    width: 8em;
-    height: 8em;
+    grid-area: img;
+    width: clamp(77px, 18vw, 6em);
+    height: clamp(77px, 18vw, 6em);
```

Comment:
ik zie je gebrukt voor de height de viewwith klopt gaat het dn nog wel goed als de scherm lager wordt? en kan je dan niet beter aspect ratio gebruken aangezien je waarde toch het zelfde zijn

Reply:
Aspect-ration is momenteel nog niet volledig [ondersteund](https://caniuse.com/mdn-css_properties_aspect-ratio). De vw werkt gewoon als normaal door de clamp css functie. De breedte en hoogte is minimaal 77px en maximaal 6em. deze waarden worden vergroot met stappen van de vw